_posts/2018-06-13-hackillinois-2019-mission.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Something old, something new..."
+date: 2018-06-15 10:00:00
+categories: recap, preview
+tags: HackIllinois2018, HackIllinois2019
+author: Kyle Begovich
+---
+
+Even though it is only June, our HackIllinois Summer team is already well underway in planning HackIllinois 2019, and we would love to offer the first look at some of our motivations and guiding principles.
+
+
+# Something Old:
+
+The blog has already explored a few retrospective pieces of the event last year, so here we will begin focusing on how it all applies to next year. We are currently in the process of engaging our mentors to get their take on their weekend on campus: what they liked and what they would like changed. These mentors are at the heart of our hacker experience, tons of fun to host, and exceptionally talented contributors in their field. If you didn’t get to engage with the mentors at the event, we hope to make this opportunity more apparent in the future.
+
+2018 also saw some very engaging workshops and lightning talks, teaching everything from Unity development to making memes. Our sponsors and students always do a great job in offering supplemental knowledge and tools that you would not otherwise have the opportunity to experience. Our sponsors are some of the biggest, best, and most cutting-edge in the tech industry, and the value they provide to the event and the attendees cannot be overstated. Being able to take a break from your project is something all of our attendees should feel comfortable doing, and we’ll continue to develop these events for the years to come.
+
+
+# Something New:
+
+There is only 8 months, 7 days, and 6 hours until check-in, but who’s counting? HackIllinois 2019 will be the 6th time Illinois students organize what has become the premier open source event available to undergraduates. While writing this statement, we wanted to explore and emphasize the varied roles that HackIllinois takes on. We have established our mission this year:
```

Comment:
"There is..." -> "There are..."
"and 6 hours..." -> "and 6 hours (at the time of writing)"
"While writing this statement" -> "While writing our mission statement"